docs/src/table_rotation.md

```diff
@@ -45,10 +45,9 @@ has read through.
 
 When table rotation is allowed, the router table uses the `curmonth`
 field to indicate the last month the client has read notifications
-through. This is independent of the last_connect since it is possible
```

Comment:
Again I can see us keeping this (and maybe the verbs to past tense) since it refers to table rotation, which we stopped doing even for DDB.

Reply:
You know, I had the same thought when I was taking it out. Given it's clearly labeled legacy, we can let it stay.